Sources/Classes/RSDeviceInfo.m

```diff
@@ -55,8 +55,8 @@ - (instancetype)init
         if (_advertisingId != nil) {
             [tempDict setValue:_advertisingId forKey:@"advertisingId"];
             [tempDict setValue:[NSNumber numberWithBool:_adTrackingEnabled] forKey:@"adTrackingEnabled"];
-            [tempDict setValue:[[NSNumber alloc] initWithInt:_attTrackingStatus] forKey:@"attTrackingStatus"];
         }
+        [tempDict setValue:[[NSNumber alloc] initWithInt:_attTrackingStatus] forKey:@"attTrackingStatus"];
```

Comment:
Check if _attTrackingStatus is Nil

Reply:
Actually _attTrackingStatus cannot be nil, as the API [here](https://github.com/rudderlabs/rudder-sdk-ios/blob/df48f964a93f75d6d6e019a45a1ef15c8a8974a7/Sources/Classes/RSContext.m#L200) is expecting an int.

And also the default value for the _attTrackingStatus is set to [RSATTNotDetermined](https://github.com/rudderlabs/rudder-sdk-ios/blob/df48f964a93f75d6d6e019a45a1ef15c8a8974a7/Sources/Classes/RSDeviceInfo.m#L36)